metadata-db-app/src/cmr/metadata_db/migrations/078_add_subscription_type_to_subscription_table.clj

```diff
@@ -0,0 +1,30 @@
+(ns cmr.metadata-db.migrations.078-add-subscription-type-to-subscription-table
+  (:require
+   [cheshire.core :as json]
+   [cmr.common.util :as util]
+   [cmr.common-app.services.ingest.subscription-common :as sub-common]
+   [config.mdb-migrate-helper :as helper]))
+
+(defn populate-new-column
+  "Pull out the Type from the matadata, then populate the new 'subscription_type' column."
```

Comment:
This docstring is obsolete. I think we can just remove this function and put the one line sql statement in the up migration function.